src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportFileUploader.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import {
+  Alert,
+  Divider,
+  message,
+  Space,
+  Typography,
+} from "antd";
+import { DragUpload } from "../../../../components/files/DragUpload";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+import { SampleMetadataImportSteps } from "./SampleMetadataImportSteps";
+
+export function SampleMetadataImportFileUploader() {
+  const { Text, Title, Paragraph } = Typography
+  const options = {
+    multiple: false,
+    showUploadList: true,
+    accept: [".xls", ".xlsx", ".csv"],
+    action: setBaseUrl(`/projects/${window.project.id}/sample-metadata/upload/file`),
+    onChange(info) {
+      const { status } = info.file;
+      if (status !== 'uploading') {
+        console.log(info.file);
+      }
+      if (status === 'done') {
+        message.success(`${info.file.name} file uploaded successfully.`);
+      } else if (status === 'error') {
+        message.error(`${info.file.name} file upload failed.`);
+      }
+    },
+  }
+  const [step, setStep] = useState(0);
+
+  return (
+    <>
+      <SampleMetadataImportSteps currentStep={step}/>
```

Comment:
Steps should be constrained by the same width as the content

Reply:
Please see 4e1622ab8b79f23d874b6b294398df63034157b4.